Documentation/contributing/development/debugging.rst

```diff
@@ -405,3 +405,48 @@ For example:
     $ make LOCKDEBUG=1
     $ # Deadlock detection during integration tests:
     $ make LOCKDEBUG=1 integration-tests
+
+CPU Profiling and Memory Leaks
+------------------------------
+
+Cilium bundles ``gops``, a standard tool for Golang applications, which
+provides the ability to collect CPU and memory profiles using ``pprof``.
+Inspecting profiles can help identify CPU bottlenecks and memory leaks.
+
+To capture a profile, use the ``cilium-bugtool`` command that is included in
```

Comment:
FYI the pprof are already captured as part of the sysdumps automatically add in https://github.com/cilium/cilium-cli/pull/1424

Reply:
I've added a mention of sysdump but opted to continue using direct `cilium-bugtool` because it's easier to point users through a smaller archive.